spark/src/main/scala/org/apache/comet/shims/ShimCometSparkSessionExtensions.scala

```diff
@@ -49,4 +49,17 @@ object ShimCometSparkSessionExtensions {
     .filter(_.isInstanceOf[Int])
     .map(_.asInstanceOf[Int])
     .headOption
+
+  def supportsExtendedExplainInfo(qe: QueryExecution): Boolean = {
+    try {
+      // Look for QueryExecution.extendedExplainInfo(scala.Function1[String, Unit], SparkPlan)
```

Comment:
Can we add clear reference here that when it is added to Spark, e.g., Spark xx+?